tests/test_api.py

```diff
@@ -34,6 +34,10 @@ class TestCourseAccessGroupsViewSet(object):
 
     url = '/course-access-groups/'
 
+    @pytest.fixture(autouse=True)
+    def setup(self, settings):
+        settings.SITE_ID = 1  # Mock get_current_site
+
```

Comment:
Very small point, but seeing a list of new identical `setup` methods in the test classes calls for a common base tests class for the `setup` method

Reply:
Makes sense, will do that!